dev/github/run_docker.sh

```diff
@@ -47,6 +47,10 @@ function main {
     run_args+=" -e ALLUXIO_MVN_TESTS=${ALLUXIO_DOCKER_MVN_TESTS}"
   fi
 
+  if [ -n "${ALLUXIO_DOCKER_MVN_PROFILES}" ]; then
```

Comment:
move new section above `ALLUXIO_DOCKER_MVN_TESTS` to maintain alphabetical order